src/streamlink/plugins/pluzz.py

```diff
@@ -140,6 +150,13 @@ def _get_streams(self):
             elif '.m3u8' in video_url:
                 for stream in HLSStream.parse_variant_playlist(self.session, video_url).items():
                     yield stream
+            elif '.mp4' in video_url:
+                match = self._mp4_bitrate_re.match(video_url)
+                if match is not None:
+                    bitrate = match.group('bitrate')
+                else:
+                    bitrate = video['format']
+                yield bitrate, HTTPStream(self.session, video_url)
```

Comment:
I think when the quality is bitrate it should have the `k` suffix. Streamlink does some conversion to bitrates so they are more comparable to resolution. Might not be a problem in this case, just an FYI.